web/src/Statusbar.tsx

```diff
@@ -44,52 +44,52 @@ type StatusBarProps = {
 }
 
 class Statusbar extends PureComponent<StatusBarProps> {
-  errorPanel(errorCount: number) {
-    let errorPanelClasses = "Statusbar-panel Statusbar-panel--error"
-    let icon = (
-      <span role="img" className="icon" aria-label="Error">
-        {errorCount > 0 ? "❌" : nbsp}
-      </span>
-    )
-    let message = (
-      <span>
-        {errorCount} {errorCount === 1 ? "Error" : "Errors"}
-      </span>
-    )
+  errorWarningPanel(errorCount: number, warningCount: number) {
     return (
-      <div className={errorPanelClasses}>
-        {icon}&nbsp;{message}
-      </div>
+      <section className="Statusbar-panel err-warn">
+        <div className="err-warn-item err-warn-item--error">
+          <ErrorSvg className={`icon ${errorCount > 0 ? "icon--error" : ""}`} />
+          <p>
+            <span className="count">{errorCount}</span> error
+            {errorCount === 1 ? "" : "s"}
+          </p>
+        </div>
+        <div className="err-warn-item">
+          <WarningSvg
+            className={`icon ${warningCount > 0 ? "icon--warning" : ""}`}
+          />
+          <p>
+            <span className="count">{warningCount}</span> warning
```

Comment:
i'm worried that having a bunch of small, unnamespaced class names like "count" are likely to become unmanageable fast. it would be better to namespace these in some way

Reply:
I agree!